scripts/build.js

```diff
@@ -24,7 +25,9 @@ var packageJsonPath = path.join(__dirname, relative, 'package.json');
 var buildPath = path.join(__dirname, relative, 'build');
 rimrafSync(buildPath);
 
-webpack(config).run(function(err, stats) {
+webpack(configServer).run(() => {});
```

Comment:
Obviously this file needs a lot of love, but the whole idea is that we need to build Client and Server independently.